### ISSUE TITLE
How to normalize the images?

### ISSUE THREAD

**Issue by RookieHong** [antoine77340/S3D_HowTo100M#10]
The images should be normalized to [0, 1], what about the mean and std? For example, the normalization for pytorch pre-trained models is: `normalize = transforms.Normalize(mean=[0.485, 0.456, 0.406],
                                 std=[0.229, 0.224, 0.225])`

**Comment by antoine77340** [comment]
The model has been trained with inputs simply scaled in [0, 1].

**Comment by RookieHong** [comment]
You mean, just simply divide the pixel values by 255?

**Comment by antoine77340** [comment]
right! it should be easier than the usual pytorch normalization :D

**Comment by RookieHong** [comment]
Thanks!
